common/dynamicconfig/configstore/config_store_client_test.go

```diff
@@ -86,7 +86,7 @@ func (s *configStoreClientSuite) SetupTest() {
 			SchemaVersion: 1,
 			Entries: []*types.DynamicConfigEntry{
 				{
-					Name: dc.Keys[dc.TestGetBoolPropertyKey],
+					Name: dc.TestGetBoolPropertyKey.String(),
```

Comment:
kinda wonder if this could be changed to hold a `Key` instead...
... but string is the same as it was, so if it's not trivial, nope.  it's fine.

Reply:
no, we cannot change it, because we store the key name in dynamic storage.